tests/embedding_lookup.cc

```diff
@@ -0,0 +1,32 @@
+#include <memory>
+#include <vector>
+
+#include "Embeddings.hh"
+#include "finalfusion.h"
+
+int main() {
+  std::unique_ptr<Embeddings> embeddings;
+  try {
+    embeddings = std::make_unique<Embeddings>("foo", true);
+    return 1;
+  } catch (std::exception &e) {
+  }
+
+  try {
+    embeddings = std::make_unique<Embeddings>("data/test.fifu", false);
+  } catch (std::exception &e) {
+    return 1;
```

Comment:
clang-format, I think?

Reply:
`clang-format embedding_lookup.cc -i` didn't change the file, so that should be fine?